packages/augur-ui/src/modules/app/components/side-nav/side-nav.tsx

```diff
@@ -78,10 +80,39 @@ const SideNav = ({
             ))}
 
             <div>
-              {showMigrateRepButton && <PrimaryButton
-                text='Migrate V1 to V2 REP'
-                action={() => migrateV1Rep()}
-              />}
+              {showMigrateRepButton && (
+                <span className={Styles.SideNavMigrateRep}>
+                  <SecondaryButton
+                    text='Migrate V1 to V2 REP'
+                    action={() => migrateV1Rep()}
+                  />
+                </span>
+              )}
+
+              {showMigrateRepButton && (
+                <span>
+                  <label
+                    className={classNames(Styles.SideNavMigrateTooltipHint)}
+                    data-tip
+                    data-for={'migrateRep'}
+                  >
+                    {helpIcon}
+                  </label>
+                  <ReactTooltip
+                    id={'migrateRep'}
+                    className={TooltipStyles.Tooltip}
+                    effect="solid"
+                    place="top"
+                    type="light"
+                  >
+                    <p>
+                      {
+                        'You have V1 REP in your wallet. Migrate it to V2 REP to use it in Augur V2'
+                      }
+                    </p>
+                  </ReactTooltip>
+                </span>
+              )}
```

Comment:
I think putting both the button and label in the same `<span>` since they have the same `showMigrateRepButton` requirement might be cleaner. This would have an added benefit of being able to use flex to align these things instead of relative positioning the icon/label

Reply:
Fixed!

Reply:
Sorry about the close then reopen. Extra spans have been removed.

![Screenshot from 2020-01-10 11-47-22](https://user-images.githubusercontent.com/1673206/72170736-67a67400-339f-11ea-8ecf-68952ae02055.png)